src/main/kotlin/no/nav/syfo/domain/Varsel.kt

```diff
@@ -24,6 +24,8 @@ data class Varsel private constructor(
 
     fun publishSvarfristExpired(): Varsel = this.copy(svarfristExpiredPublishedAt = nowUTC())
 
+    fun isExpired(): Boolean = svarfrist.isBefore(LocalDate.now()) || svarfrist.isEqual(LocalDate.now())
```

Comment:
`isBeforeOrEqual()` finnes ikke?

Reply:
Nei, måtte eventuelt laget en extension function for det :man_shrugging: